apps/prairielearn/src/lib/code-caller/code-caller-container.ts

```diff
@@ -148,7 +167,7 @@ export class CodeCallerContainer {
     this.coursePath = null;
     this.forbiddenModules = [];
 
-    this._checkState();
+    this._checkState(undefined);
```

Comment:
This is kinda weird. If you change the `_checkState` type to `allowedStates?: CallerState[]`, you don't need to explicitly specify this argument.